app/src/dev/res/xml/shortcuts.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <shortcut
+        android:shortcutId="bookmark"
+        android:enabled="true"
+        android:icon="@mipmap/ic_launcher"
+        android:shortcutShortLabel="@string/bookmark"
+        android:shortcutLongLabel="@string/bookmark">
+
+        <intent
+            android:action="android.intent.action.VIEW"
+            android:targetPackage="com.cvtracker.vmd2.dev"
+            android:targetClass="com.cvtracker.vmd.bookmark.BookmarkActivity"
```

Comment:
As we support "https://vitemadose.convidtracker.fr/bookmarks" url, it should work targeting `MainActivity` with data `https://vitemadose.convidtracker.fr/bookmarks` right ?
Then we would have the `MainActivity` below `BookmarkActivity`

Reply:
I didn't see `MainActivity.onNewIntent(…)` and deeplink.
I update with e36ef30